spec/helpers/authentication_helper_spec.rb

```diff
@@ -38,6 +38,29 @@
     end
   end
 
+  describe "#provider_keys_configured?(provider)" do
+    provider = "facebook"
```

Comment:
Let's use rspec's `let` [helper method](https://relishapp.com/rspec/rspec-core/v/2-10/docs/helper-methods/let-and-let) here, which is more idiomatic :)
```suggestion
    let(:provider) { "facebook" }
```